File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/TreeViews_ViewMyTreeViewElementsRow.cpp

#include "TreeViews_ViewMyTreeViewElementsRow.h"
#include "TreeViews_ViewMyTreeViewElementsRow.h"
#include <string>

namespace widgetassertions
{
  bool TreeViews_ViewMyTreeViewElementsRow::getIsIncludeCheckBoxChecked( ) 
  {
    return this->isIncludeCheckBoxChecked;
  }
  std::string TreeViews_ViewMyTreeViewElementsRow::getNameLabelText( ) 
  {
    return this->NameLabelText;
  }
  std::string TreeViews_ViewMyTreeViewElementsRow::getDescriptionLabelText( ) 
  {
    return this->DescriptionLabelText;
  }
  std::string TreeViews_ViewMyTreeViewElementsRow::getInfoImageName( ) 
  {
    return this->InfoImageName;
  }
  int TreeViews_ViewMyTreeViewElementsRow::getRowIndex( ) 
  {
    return this->RowIndex;
  }
  int TreeViews_ViewMyTreeViewElementsRow::getRowDepth( ) 
  {
    return this->RowDepth;
  }
}

File: solutions/de.vimotest.sandbox/source_gen/de/vimotest/sandbox/widgetassertions/TreeViews_ViewMyTreeViewElementsRow.cs
/// <filename>
///     TreeViews_ViewMyTreeViewElementsRow.cs
/// </filename>


namespace widgetassertions
{
    internal class TreeViews_ViewMyTreeViewElementsRow
  {
        private bool isIncludeCheckBoxChecked;
     
        public     virtual bool getIsIncludeCheckBoxChecked()
    {
      return this.isIncludeCheckBoxChecked;
    }
     
        private string NameLabelText;
     
        public     virtual string getNameLabelText()
    {
      return this.NameLabelText;
    }
     
        private string DescriptionLabelText;
     
        public     virtual string getDescriptionLabelText()
    {
      return this.DescriptionLabelText;
    }
     
        private string InfoImageName;
     
        public     virtual string getInfoImageName()
    {
      return this.InfoImageName;
    }
     
        private int RowIndex;
     
        public     virtual int getRowIndex()
    {
      return this.RowIndex;
    }
     
        private int RowDepth;
     
        public     virtual int getRowDepth()
    {
      return this.RowDepth;
    }
  }
}

